spec.yaml

```
@@ -348,15 +348,18 @@ components:
               name:
                 type: string
                 description: >
-                  A supported resource type (file, folder, calendar, contact,
-                  ...).
-                  Implementations MUST support `file` at a minimum.
+                  A supported resource type (file, folder, calendar, contact, ...). 
```

The formatting was adjusted to fit 72 chars per line per IETF constraint. We could actually add a linting action to check for that.
```suggestion
                  A supported resource type (file, calendar, contact, ...). 
```